kratos/solving_strategies/convergencecriterias/convergence_criteria.h

```diff
@@ -371,6 +377,20 @@ class ConvergenceCriteria
         KRATOS_CATCH("");
     }
 
+    /**
+     * @brief This method provides the defaults parameters to avoid conflicts between the different constructors
+     * @return The default parameters
+     */
+    virtual const Parameters GetDefaultParameters() const
```

Comment:
```suggestion
    virtual Parameters GetDefaultSettings() const
```

in python its called default settings

Reply:
Not in Processes... (this is the problem when creating new methods without checking what has been done previously)

Reply:
the python was there first 
also c++ can be changed easier bcs it will just not compile :)